.github/workflows/ci.yml

```diff
@@ -0,0 +1,92 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    env:
+      VERBOSE: 1
+      GOFLAGS: -mod=readonly
+
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build
+        run: make build
+
+  # TODO: Replace with Action
+  # https://github.com/golangci/golangci-lint-action
```

Comment:
My 2 cents: Doesn't make much of a difference. I usually try to stick to the same tools in CI and local development.